src/sobre.md

```diff
@@ -23,21 +27,18 @@ implica em obrigatoriedade de atividades**, e **não existe hierarquia que
 "escale membros"** para projetos.
 
 ## Como participar?
-Não  temos um processo seletivo estrito, basta entrar em um dos nossos
-grupinhos e dizer "oi". Te explicaremos sobre os projetos em andamento e as
-nossas reuniões. Caso você participe de alguma maneira, você pode (e deve!) se
-considerar um membro!
 
-Basta entrar em um dos nossos canais ([telegram](https://telegram.gelos.club)
-ou [matrix](https://matrix.gelos.club)) e você está dentro!
```

Comment:
Não tem mais o Matrix?

Reply:
Tem sim, ainda tem link no nosso footer. Mas achei melhor deixar só o telegram aqui, pq:
1 - A experiencia no matrix tá meio miseravel pq n tem os topicos
2 - São esses primeiros parágrafos que sites de notícia vão pegar pra reproduzir. Então acho melhor que eles divulguem só o telegram, pra ficar mais claro.